### PR TITLE
103 incorrect reference to error in stdauth middleware

### PR DESCRIPTION
**Pull request by Joao-Potes**

### CODE REVIEW

File: middlewares/user.middleware.js [hunk 1]
```diff
@@ -38,9 +38,17 @@ export const stdAuth = async (req, res, next) => {
...
+                message: 'Session expired! Please login again.',
+                redirect: '/login.html',
+            });
+        }
+        if (e instanceof jwt.MalformedTokenError) {
+            return res.status(401).json({
+                message: 'Malformed token! Please login again.',
+                redirect: '/login.html',
+            });
         }
```
Comment (TomStrong) [inline middlewares/user.middleware.js:52]: need to log the error

**Review by TomStrong — CHANGES_REQUESTED** [review]
Good work, just one change

**Comment by TomStrong** [comment]
Probably worth adding this code to the admin auth middleware too

**Review by ali1373107 — APPROVED** [review]
reviewed

**Pull request merged**